numpy/core/tests/test_umath.py

```diff
@@ -926,7 +931,7 @@ def test_log_values(self):
             assert_raises(FloatingPointError, np.log, np.float32(-np.inf))
             assert_raises(FloatingPointError, np.log, np.float32(-1.0))
 
-        # See https://github.com/numpy/numpy/issues/18005 
+        # See https://github.com/numpy/numpy/issues/18005
```

Comment:
My .vimrc autocorrected this, can undo and rebase if required, please let me know